.github/workflows/scripts/spdx-tools-java-wrapper.sh

```diff
@@ -1,47 +1,49 @@
 #!/usr/bin/env bash
 
 # SPDX-FileCopyrightText: Maximilian Huber <maximilian.huber@tngtech.com>
+# SPDX-FileCopyrightText: Sebastian Schuberth <sschuberth@gmail.com>
 #
 # SPDX-License-Identifier: CC0-1.0
 
-set -euo pipefail
+set -eo pipefail
```

Comment:
I was playing with the idea to use a default value, but that would make it hared to `echo "Writing local job summary to '$GITHUB_STEP_SUMMARY'."` only if `GITHUB_STEP_SUMMARY` was unset.

Reply:
but `$GITHUB_STEP_SUMMARY` is no issue here in your implementation.

Reply:
I'm not sure if we're on the same page. My point is: A `[ -z "$GITHUB_STEP_SUMMARY" ]` with `set -u` fails with `line 32: GITHUB_STEP_SUMMARY: unbound variable` (which is kind of stupid, as I'm explicitly checking for it to be set). But working around that by using a default value would require me to write an extra condition for the `echo` call.

Reply:
Oh sorry, I was wrong here. Following https://stackoverflow.com/a/13864829 the right way is
```
[ -z "${GITHUB_STEP_SUMMARY+x}" ]
```
> where ${var+x} is a [parameter expansion](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02) which evaluates to nothing if var is unset, and substitutes the string x otherwise.

Reply:
I've found a work-around that I can live with.